internal/store/static.go

```diff
@@ -114,11 +114,16 @@ func (store *StaticStore) GetPlugins(pluginFilter *model.PluginFilter) ([]*model
 }
 
 // getPlugins returns all plugins compatible with the given server version, sorted by name ascending.
-func (store *StaticStore) getPlugins(serverVersion string) ([]*model.Plugin, error) {
+// Only returns enterprise plugins, if enterprisePlugins is true.
```

Comment:
I initially read this as, "if enterprisePlugins is true, only return enterprise plugins", but of course that's not the semantics. Could we just rename the variable to `includeEnterprisePlugins` and eliminate this comment (and it's ambiguity)?

Reply:
Good suggestion :+1: